usr/share/bash-completion/completions/kapacitor

```diff
@@ -165,6 +164,38 @@ _kapacitor()
     return 0
 }
 
+_kapacitor_files_add_postfix()
+{
+	#Add a slash (directories) or space (files except directories) after name
+	local c i=0
+	for c in "${COMPREPLY[@]}"; do
+		if [ -d "${c}" ]; then
+			COMPREPLY[i++]="${c}/";
+		else
+			COMPREPLY[i++]="${c} ";
+		fi
+	done
+        compopt -o nospace
```

Comment:
OK, so why is it after the loop? Looks like its only set after compgen has already been called.